src/main/java/se/sundsvall/myrepresentative/api/model/mandates/Mandate.java

```diff
@@ -44,7 +46,7 @@ public static class Permission {
         @Schema(description = "Code for the specific permission", example = "bf1a690b-33d6-4a3e-b407-e7346fa1c97c")
         private String code;
 
-        @Schema(description = "UUID for the mandate", example = "95189b70-c0cc-432f-a1ef-bb75b876ab75")
-        private String mandate;
+        @Schema(description = "Description for the specific permission", example = "Fullmakt för att hantera ansökan om standskyddsdispens")
```

Comment:
strandskyddsdispens?

Reply:
ståndskyddsdispens?

Reply:
Not sure who made the typo. I just copied. Might be the actual value from bolagsverket :D <img width="339" alt="image" src="https://github.com/Sundsvallskommun/api-service-my-representative/assets/110965999/51f87884-3ae4-466f-8fdf-29ddc43334e5">